Extensions/Today/TodayViewController.swift

```diff
@@ -106,38 +114,50 @@ class TodayViewController: UIViewController, NCWidgetProviding, TodayWidgetAppea
 
     func adjustFonts() {
         if traitCollection.preferredContentSizeCategory >= .accessibilityMedium {
-            newTabButton.label.font = newTabButton.label.font.withSize(26)
-            newPrivateTabButton.label.font = newPrivateTabButton.label.font.withSize(26)
-            openCopiedLinkButton.label.font = openCopiedLinkButton.label.font.withSize(26)
-        }
-        else if traitCollection.preferredContentSizeCategory <= .large {
-            newTabButton.label.font = newTabButton.label.font.withSize(14)
-            newPrivateTabButton.label.font = newPrivateTabButton.label.font.withSize(14)
-            openCopiedLinkButton.label.font = openCopiedLinkButton.label.font.withSize(14)
+            resize(size: 25)
+        } else if traitCollection.preferredContentSizeCategory <= .extraExtraExtraLarge && traitCollection.preferredContentSizeCategory > .extraLarge {
```

Comment:
nit: Wondering it it'll be possible to convert into switch 🤔 
@noorhashem any thoughts on if we can improve this?